src/main/scala/com/microsoft/partnercatalyst/fortis/spark/analyzer/timeseries/ComputedTilesToComputedTrends.scala

```diff
@@ -0,0 +1,45 @@
+package com.microsoft.partnercatalyst.fortis.spark.analyzer.timeseries
+
+import com.microsoft.partnercatalyst.fortis.spark.dto.{ComputedTile, ComputedTrend}
+import org.apache.commons.math3.stat.regression.SimpleRegression
+import org.apache.spark.rdd.RDD
+
+object ComputedTilesToComputedTrends {
+
+  def apply(computedTiles: RDD[ComputedTile]): RDD[ComputedTrend] = {
+    val groupedTiles = computedTiles.groupBy(tileGroup)
+    groupedTiles.flatMap[ComputedTrend]((group) => {
+      val tiles = group._2.toSeq
+
+      val regressionResult = linearRegression(tiles)
+
+      val score = regressionResult.getSlope
+      if (score.isNaN) Seq[ComputedTrend]()
+      else {
+        val latestTile = tiles.sortBy(t => -t.periodstartdate).head
+        val filteredTiles = tiles.filter(t => t.period == latestTile.period)
+        val trends = filteredTiles.map(new ComputedTrend(_, score))
```

Comment:
I'm not sure what we're doing here. From my understanding, the idea for the anomaly detection was to fit a linear model to the mention counts and then flag any data point that has a high error with regards to the model as an anomaly, i.e. trend. The code here instead fits the regression, discards its intercept and tags every data point with the slope of the overall regression. What will this value be used for?